backend-api/src/functions/asyncActiveSession/jwe/jweDecrypter.ts

```diff
@@ -57,13 +57,13 @@ export class JweDecrypter implements IDecryptJwe {
     } catch (error) {
       return errorResult({
         errorMessage: `Error decrypting JWE: Unable to decrypt encryption key: ${error}`,
-        errorCategory: "CLIENT_ERROR",
+        errorCategory: "SERVER_ERROR",
```

Comment:
We need to look into this. Right now we are saying that ALL KMS decryption errors are server errors, and I think we might also want to return a client error depending on the exception type. That's something for the next ticket.